br/pkg/restore/snap_client/pipeline_items.go

```diff
@@ -367,6 +367,75 @@ func (buffer *statsMetaItemBuffer) TryUpdateMetas(ctx context.Context, statsHand
 	return statsHandler.SaveMetaToStorage("br restore", false, metaUpdates...)
 }
 
+func calculateRowCountForPhysicalTable(files []*backuppb.File, availableIndexCount int) int64 {
+	totalKvs := uint64(0)
+	for _, file := range files {
+		totalKvs += file.TotalKvs
+	}
+	return int64(totalKvs / uint64(availableIndexCount+1))
```

Comment:
If the `no_null_index` feature comes in, this won't work. cc @bb7133

Reply:
@Leavrth is it a must to make the table row count accurate?

Reply:
After analyze, it will update the real row count.